content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Add a new Custom Ingress Gateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/setup/).
+* Setup `cert-manager` with helm [chart](https://github.com/helm/charts/tree/master/stable/cert-manager#installing-the-chart)
+
+## Configuring the custom ingress gateway
+
+1. Check if [cert-manager](https://github.com/helm/charts/tree/master/stable/cert-manager) was installed using Helm with the following command:
+
+    {{< text bash >}}
+
+    $ helm ls
+
+    {{< /text>}}
+
+    The output should be similar to the example below and show cert-manager with a `STATUS` of `DEPLOYED`:
+
+    {{< text plaintext >}}
+
+    NAME   REVISION UPDATED                  STATUS   CHART                     APP VERSION   NAMESPACE
+    istio     1     Thu Oct 11 13:34:24 2018 DEPLOYED istio-1.0.2               1.0.2         istio-system
+    cert      1     Wed Oct 24 14:08:36 2018 DEPLOYED cert-manager-v0.6.0-dev.2 v0.6.0-dev.2  istio-system
+
+    {{< /text >}}
+
+1. To create the cluster's issuer, apply the following configuration:
+
+    {{< warning_icon >}} Change the cluster's [issuer](https://cert-manager.readthedocs.io/en/latest/reference/issuers.html#issuers) provider with your own configuration values. The example uses the values under `route53`.
+
+    {{< text yaml >}}
+
+    apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
I am in favor of using commands with embedded configurations wherever possible. See for example https://preliminary.istio.io/blog/2018/egress-tcp/#mesh-external-service-entry-for-an-external-mysql-instance.

The reader should be able to follow the task with as less clicks as possible. If editing a config is really necessary, I would create the config file for user, see for example https://preliminary.istio.io/docs/examples/advanced-gateways/wildcard-https-egress-gateway/#custom-egress-gateway-with-sni-proxy, and then add a command to deploy that file, and a command to remove that file.